templates/prometheus.yaml

```diff
@@ -81,6 +81,12 @@ spec:
   ruleNamespaceSelector:
     matchExpressions:
       - { key: "monitoring-key", operator: In, values: [{{ .MonitoringKey }}] }
+  podMonitorNamespaceSelector:
+    matchExpressions:
+      - { key: "monitoring-key", operator: In, values: [{{ .MonitoringKey }}] }
+  podMonitorSelector:
```

Comment:
I think David was thinking about only using namespace selectors and always selecting all resources in that namespace. Can you confirm @david-martin ? Should we still have the resource selector?

Reply:
Correct, we can just add the namespace selector here.
(cc @obrienrobert who is looking at removing the serviceMonitorSelector & ruleSelector)

Reply:
ok, I will remove it then